### DISCUSSION TITLE
Once I click Prev action from 2nd step in Product tour to parent step in useRoutes react-router v6, it skips first step and goes to 2nd step.

### DISCUSSION THREAD

**Discussion by CosmicTiger** [gilbarbara/react-joyride#905] [Q&A]
Hello, I'm kinda of an issue here. I have my code practically as the one followed in the documentation from react-joyride/multiroutes demo and basically I'm unable to click previous step from my route A to Home route because it skips directly to route A once again instead of staying at Home as similar in the demo. Is there's a way to stop the callback from not doing the step:after action once the component is not kinda loaded? I'm guessing could be because I have Suspense and Lazy Loading implemented in the routes due to the data that I'm fetching individually in each route.

**Comment by gilbarbara** [comment]
Without knowing how you are triggering the steps is nearly impossible to tell what's happening.
Please share a minimal reproducible example.

That said, you can't change the step lifecycle.
Does the Home route have an automatic trigger to advance to the next step?
In the demo, the call to advance to the next step is a click handler so that it won't run automatically.
